packages/terra-tabs/src/ModularTabs.module.scss

```diff
@@ -57,11 +57,11 @@
 
```

Comment:
Changes to the modular tabs scss file isn't necessary, it's dead code.
As a side note: Weren't we going to delete the modular tab code?

Reply:
Yeah we can do that, but it can be done on a later pull request.